test/test-rewrite-url.js

```diff
@@ -0,0 +1,177 @@
+var assert = require('assert');
+var _ = require('@sailshq/lodash');
+var async = require('async');
+var request = require('request');
+var fs = require('fs');
+
+describe('Apostrophe Sitemap: test rewrite URL', function() {
+
+  var apos;
+
+  this.timeout(5000);
+
+  after(function(done) {
+    try {
+      require('apostrophe/test-lib/util').destroy(apos, done);
+    } catch (e) {
+      console.warn('Old version of apostrophe does not export test-lib/util library, just dropping old test db');
+      apos.db.dropDatabase();
+      setTimeout(done, 1000);
+    }
+  });
+
+  it('should be a property of the apos object', function(done) {
```

Comment:
Since this test (and all the others except for the last one) doesn't test anything but just instantiate apos object (or insert data for the next ones) for tests purpose, I would have put this code in the `before` hook to be semantically more correct.